README.md

```diff
@@ -74,6 +74,32 @@ To use React UI in your app:
    );
    ```
 
+### Development mode
+
+It is possible to run React UI in your app in development mode. This is useful because:
+
+- In production mode React UI uses non-human-readable class names.
+- Runtime check (i.e. PropTypes) are suppressed in production mode.
+
+To enable development mode add to the `webpack.config.js` file in your app:
+
+```js
+module.exports = (env, argv) => ({
+  /* … */
+  resolve: {
+    alias: {
+      // Allow to run react-ui in development mode for easier development
+      '@react-ui-org/react-ui$': argv.mode === 'production'
+        ? '@react-ui-org/react-ui/dist/lib.js'
+        : '@react-ui-org/react-ui/dist/lib.development.js',
+    }
+  }
+  /* … */
+});
+```
+
+Then run webpack with the `--mode=production` or `--mode=development`flag.
```

Comment:
Please improve this either to Webpack (capital W, it's a name) or `webpack` (as a command).

Reply:
Please see: https://webpack.js.org/concepts/, the first paragraph. I was not sure how to write it, so I went to take a look there.